x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -9,17 +9,35 @@ export const PACKAGES_SAVED_OBJECT_TYPE = 'epm-packages';
 export const ASSETS_SAVED_OBJECT_TYPE = 'epm-packages-assets';
 export const MAX_TIME_COMPLETE_INSTALL = 60000;
 
+export const FLEET_SYSTEM_PACKAGE = 'system';
+export const FLEET_ELASTIC_AGENT_PACKAGE = 'elastic_agent';
 export const FLEET_SERVER_PACKAGE = 'fleet_server';
+export const FLEET_ENDPOINT_PACKAGE = 'endpoint';
 
-export const requiredPackages = {
-  System: 'system',
-  Endpoint: 'endpoint',
-  ElasticAgent: 'elastic_agent',
-  FleetServer: FLEET_SERVER_PACKAGE,
-} as const;
+/*
+ Package rules:
```

Comment:
Thanks for the table! Does `requiredPackages` just mean that they are not removable? I think that's what I'm seeing from the code. Maybe we should change it to `unremovable` or something. `required` makes me think that it must be installed by just reading the name. The table helps clarify it though 😄